### DISCUSSION TITLE
Wpcli can't access mysql executable between containers

### DISCUSSION THREAD

**Discussion by AdsonCicilioti** [wp-cli/wp-cli#5671] [How do I?]
Hy guys!

I'm trying run db commands between containers. I have 3 containers (wp / nginx / mysql). All works fine with de wpcli executable placed into wp root.

Commands to manage `themes`, `plugins` and the `core`, woks finely, but when i try to run an `db import sql` command, i receive the follow error message:
```
Error: Failed to get current SQL modes. Reason: env: can't execute 'mysql': No such file or directory
```

Seems because `wpcli` executable don't see the `PATH` of `mysql` container.
I know this it is a Docker/Podman/Containers question, but i would like to know if anybody has any experience with this scenario, or can pointing a way to solve this.

**Comment by ecreationstudio** [comment]
Hey @AdsonCicilioti,

have you already solved the problem?
